website/content/docs/release-notes/1.9.0.mdx

```diff
@@ -0,0 +1,119 @@
+---
+layout: docs
+page_title: 1.9.0
+description: |-
+  This page contains release notes for Vault 1.9.0.
+---
+
+# Vault 1.9.0 Release notes
+
+**Software Release Date**: November 19, 2021
+
+**Summary**: This document captures major updates as part of Vault release 1.9.0, including new features, breaking changes, enhancements, deprecation, and EOL plans. Refer to the [Changelog](https://github.com/hashicorp/vault/blob/main/CHANGELOG.md) for additional changes made within the Vault 1.9 release.
+
+## New Features
+
+This section describes the new features introduced as part of Vault 1.9.0.
+
+### Client Count Improvements
+
+Several improvements to client count were made to help customers better track and identify client attribution and reduce overcomputing.
+
+#### Improved computation of client counts and usability within the usage metrics UI
+
+The improvements made include the following:
+
+* New logic enables de-duplication of non-entity tokens, thereby reducing their contribution towards the client count
+* New logic allows entities to be created for local auth mounts, thereby eliminating  non-entity-tokens being  issued by the local auth mounts and reducing the overall client count
+* Eliminates root tokesn from the client count aggregate
+* Displays client counts per namespace (top ten, descending order by attribution) in the usage metrics UI with the ability to export data for all namespaces
+* Displays clients earlier than a month in the usage metrics UI (within ten minutes since initiation of computation)
+
+### Advanced Data Protection Module (ADP) Enhancements
+
+The following section provides details about the ADP module features added in this release.
+
+#### Advanced I/O handling for Tranform FPE (ADP-Transform)
+
+Users of the Format Preserving Encryption (FPE) feature of ADP Transform will now benefit from increased flexibility with regards to formatting the input and output of their data. [Transformation templates](https://learn.hashicorp.com/tutorials/vault/transform#advanced-handling) are receiving two new fields- **encode_format** and **decode_formats** -that allow users to specify and format individual [capturing groups](https://www.regular-expressions.info/refcapture.html) within the regular expressions that define their formats.
+
+#### MS SQL TDE (ADP-KM)
+
+We added support to Vault Enterprise for customers who want Vault to manage encryption keys for Transparent Data Encryption on MSSQL servers.
+
+#### Key Management Secrets(KMS) engine - GCP (ADP-KM)
+
+The [KMS Engine for GCP](https://www.vaultproject.io/docs/secrets/gcpkms) provides key management via the Google Cloud KMS to assist with automating many GCP key management functions. This was introduced as a Tech Preview feature in Vault 1.8 and is now being supported as a generally available feature.
```

Comment:
I think this is still incorrect. The feature was never released as a tech preview in 1.8. GCP KMS support for key management is brand new in 1.9.

Reply:
Thanks @austingebauer . The text was provided by PM, but I'll go ahead and make the change and will circle back w/ them to let them know it was never released as a tech preview. Thanks for your feedback.

Reply:
Thank you, @taoism4504 👍

Reply:
Fixed